src/zabapgit_persistence.prog.abap

```diff
@@ -1562,6 +1584,15 @@ CLASS lcl_persistence_settings IMPLEMENTATION.
       CATCH lcx_not_found.
         ro_settings->set_run_critical_tests( abap_false ).
     ENDTRY.
+    TRY.
+        lv_max_lines_as_string = lcl_app=>db( )->read(
+           iv_type  = 'SETTINGS'
+           iv_value = 'MAX_LINES' ).
+        lv_max_lines_as_integer = lv_max_lines_as_string.
+        ro_settings->set_max_lines( lv_max_lines_as_integer ).
+      CATCH lcx_not_found.
+        ro_settings->set_max_lines( 0 ).
```

Comment:
I guess this is where the default is set? Can we set it to 500? That way abapGit will not seem unresponsive to a new user trying a repository with 25k objects 😄